src/store/modules/getters/ui.js

```diff
@@ -1,7 +1,12 @@
 /* eslint import/prefer-default-export: "off" */
+const ErrorType = { INSUFFICIENT_BALANCE: 'INSUFFICIENT_BALANCE' };
 
 export const getLocale = state => state.locale;
 export const getAvailableLocales = state => state.locales;
 
 export const getIsSlidingMenuOpen = state => state.isSlidingMenuOpen;
 export const getIsHK = state => state.isHK;
+
+export const getIsOpenSnackbar = state => state.isOpenSnackbar;
+export const getAlertType = state => state.alertType;
+export const getAlertMessage = state => state.alertMessage;
```

Comment:
I suggest that do not add prefix `get` for getters anymore, maybe just `ui`, e.g. `uiAlertType` instead of `getAlertType`